MicrosoftDX/nether
# Gamertag fix

### Issue: #354 


### Description:
Fix the player store to prevent re-using a gamertag or changing a gamertag
Update the implicit flow UI to show friendly message when gamertag is already in use

### Test:
* Configure facebook sign in as per [docs](https://github.com/MicrosoftDX/nether/blob/master/documentation/identity/configuration.md#facebook)
* Build and run Nether.Web
* Run integration tests
* Remove your user/userlogin entry from the database
* Add a Player to the database with gamertag `testtag`
* Open swagger UI and go through the authorise flow, logging in with Facebook. When asked for gamertag enter `testtag` and verify that you get a message displayed indicating that the gamertag is already in use
